aws-rds-dbcluster/src/test/java/software/amazon/rds/dbcluster/CreateHandlerTest.java

```diff
@@ -481,4 +488,30 @@ public void handleRequest_CreateDbCluster_DomainNotFoundException() {
 
         verify(rdsProxy.client(), times(1)).createDBCluster(any(CreateDbClusterRequest.class));
     }
+
+    @Test
+    public void CreateHandlerConfig_Delay_MatchesExpectedIntervals()
+    {
+        expectServiceInvocation = false;
+        final CreateHandler handler = new CreateHandler();
+        final Delay delay = handler.config.getBackoff();
+
+        final List<Duration> expectedIntervalsBetweenInvocations = ImmutableList.of(
+                Duration.ofSeconds(1),
+                Duration.ofSeconds(3),
+                Duration.ofSeconds(7),
+                Duration.ofSeconds(13),
+                Duration.ofSeconds(21),
+                Duration.ofSeconds(31),
+                Duration.ofSeconds(43),
+                Duration.ofSeconds(57),
+                Duration.ofMinutes(1).plus(Duration.ofSeconds(13)),
+                Duration.ofMinutes(1).plus(Duration.ofSeconds(31))
+        );
+
+        for (int i = 0; i < expectedIntervalsBetweenInvocations.size(); i++) {
+            Assertions.assertThat(delay.nextDelay(i+1)) // attempt number starts at 1
+                    .isEqualTo(expectedIntervalsBetweenInvocations.get(i));
+        }
```

Comment:
Frankly, I don't see much value in this test: any config change turns it into a time calculation burden. The Delay class is tested well enough, we can assume it simply works.